File: 30-days-of-code/csharp/Day1.cs
using System;

namespace Day1
{
    class Day1
    {
        static void Main(string[] args)
        {
            // initialisation
            int i = 4;
            double d = 4.0;
            string s = "HackerRank ";

            // read input
            int i2 = Convert.ToInt32(Console.ReadLine());
            double d2 = Convert.ToDouble(Console.ReadLine());
            string s2 = Console.ReadLine();

            // print sum statement
            Console.WriteLine(i + i2);
            Console.WriteLine(String.Format("{0:N1}", d + d2));
            Console.WriteLine(s + s2);
        }
    }
}

File: 30-days-of-code/java/Day1.java
import java.util.*;

class Day1 {
    public static void main(String[] args) {
        // initialisation
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";

        // create scanner
        Scanner in = new Scanner(System.in);

        // read input
        int i2 = in.nextInt();
        double d2 = in.nextDouble();
        // skip line
        in.nextLine();
        String s2 = in.nextLine();

        // close scanner
        in.close();

        // print sum statement
        System.out.println(i + i2);
        System.out.println(d + d2);
        System.out.println(s + s2);
    }
}
